lib/sensu-plugin/utils.rb

```diff
@@ -3,9 +3,9 @@ module Plugin
     module Utils
 
       def config_files
-        if ENV['SENSU_LOADED_TEMPFILE']
+        if ENV['SENSU_LOADED_TEMPFILE'] && File.file?(ENV['SENSU_LOADED_TEMPFILE'])
           IO.read(ENV['SENSU_LOADED_TEMPFILE']).split(':')
-        elsif ENV['SENSU_CONFIG_FILES']
+        elsif ENV['SENSU_CONFIG_FILES'] && File.file?(ENV['SENSU_CONFIG_FILES'])
```

Comment:
I wonder if testing for the existence of `ENV['SENSU_CONFIG_FILES']` is appropriate here? At this point it is a colon separated list of many files, so I'd expect this would always evaluate to false.